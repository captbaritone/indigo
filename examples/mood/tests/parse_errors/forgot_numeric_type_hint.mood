pub fn test(): i32 {
  42
}