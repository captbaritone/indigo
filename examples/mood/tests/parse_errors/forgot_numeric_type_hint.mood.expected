-----------------
INPUT
----------------- 
pub fn test(): i32 {
  42
}
-----------------
OUTPUT
-----------------
Error: Expected a number literal to be followed by a type hint.:
 --> forgot_numeric_type_hint.mood:3:1

  |   42
3 | }
  | ^ Expected a numeric type hint like "_i32" or "_f32"
